README.md

```diff
@@ -66,7 +75,7 @@ Usage of ./endlessh-go
   -logbuflevel int
         Buffer log messages logged at this level or lower (-1 means don't buffer; 0 means buffer INFO only; ...). Has limited applicability on non-prod platforms.
   -logtostderr
-        log to standard error instead of files
+        Log to standard error instead of files
```

Comment:
You should not worry about these.

They are from `./endlessh-go --help` Even if you are changing it now, next time I copy the output of `./endlessh-go --help` will overwrite them again.

If you do want to change text, you need to change at where the flag declared.